src/Comparator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace Roave\ApiCompare;
+
+use Roave\BetterReflection\Reflector\ClassReflector;
+use Roave\BetterReflection\Reflector\Exception\IdentifierNotFound;
+
+final class Comparator
+{
+    public function compare(ClassReflector $oldApi, ClassReflector $newApi): array
+    {
+        $changelog = [];
+
+        foreach ($oldApi->getAllClasses() as $oldClass) {
+            try {
+                $newClass = $newApi->reflect($oldClass->getName());
```

Comment:
This approach only looks for BC breaks, not API additions

Reply:
As I understand, this one is to look for public API removal, see `catch`.